godtools/App/DependencyContainer/AppDataLayerDependencies.swift

```diff
@@ -165,6 +165,14 @@ class AppDataLayerDependencies {
         )
     }
     
+    func getLessonListItemProgressRepository() -> GetLessonListItemProgressRepository {
```

Comment:
Hey @rachaelblue this dependency I'd like moved into the UserLessonProgressDataLayerDependencies.  I'm wondering if that's what you intended, but out of habit moved into here?  I know the specific feature dependency containers is something new we've been transitioning to.

Reply:
Hey @rachaelblue I'm going to mark this resolved since this is shared across features.